src/monitoring_tests/cost_coverage_zero_signed_fee.py

```diff
@@ -37,20 +36,12 @@ def cost_coverage(self, competition_data: dict[str, Any], gas_cost: float) -> bo
         orders = solution["orders"]
         native_prices = competition_data["auction"]["prices"]
         total_fee = 0.0
-        zero_signed_fee_market = False
```

Comment:
This was removed since all orders are zero signed fees, right? Maybe the test can be renamed then to just cost coverage.

Reply:
Yeah good point. Let me try to clean it up